CMakeLists.txt

```diff
@@ -263,6 +262,10 @@ if ( NOT ${WRFIO_NCD_NO_LARGE_FILE_SUPPORT} )
   message( STATUS "netCDF large file support not suppressed, if available it will be used" )
 endif()
 
+# Hydro option requires MPI
+if ( ENABLE_HYDRO )
```

Comment:
```suggestion
if ( ${ENABLE_HYDRO} AND NOT ${USE_MPI} )
  message( STATUS "WRF Hydro requires MPI usage" )
```

Reply:
I forgot to mark that deviations from user input/defaults should probably have an output message associated with it.

Reply:
Committed this, thanks again for the reviews!